manifests/init.pp

```diff
@@ -125,6 +129,16 @@
 
   include 'aide::default_rules'
 
+  if $rules =~ Hash {
+    $rules.each |String $key, Hash $attrs| {
```

Comment:
I could remove this, but this is what I might expect a parameter called `rules` to do given that their is a `aide::rule` defined type.

Reply:
Yeah, this is a good expected use case.